src/main/webapp/blobstoreFunctions.js

```diff
@@ -0,0 +1,38 @@
+/** Loads action URL and attaches it to the upload form (the user's browser uploads the file to blobstore with the generated url) */
+function fetchBlobstoreUrl() {
+  fetch('/blobstore-upload-url').then((response) => response.text()).then((imageUploadUrl) => {
+    document.getElementById('plant-pic-form').action = imageUploadUrl;
+  });
+}
+ 
+/**
+  * Creates and adds a card element for the photo 
+  * pic_url is the URL of image to be shown in the post
+*/
+function createListElement(pic_url) {
```

Comment:
create list element or create card element?

Reply:
I changed it to create card element